l1-contracts/contracts/bridge/L1AssetRouter.sol

```diff
@@ -108,12 +53,21 @@ contract L1AssetRouter is IL1AssetRouter, ReentrancyGuard, Ownable2StepUpgradeab
     /// @dev Address of native token vault.
     IL1NativeTokenVault public nativeTokenVault;
 
+    /// @dev Address of l1 nullifier.
+    IL1Nullifier public l1Nullifier;
```

Comment:
We will want to deploy the AR on the L2 as well, so the name should be something like NullifierStorage